src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/FrameBufferPointerInputSource.Mouse.cs

```diff
@@ -0,0 +1,126 @@
+		var properties = new PointerPointProperties();
+		Action<PointerEventArgs>? raisePointerEvent = null;
+
+		if (type == LIBINPUT_EVENT_POINTER_MOTION_ABSOLUTE)
+		{
+			_mousePosition = new Point(
+				x: libinput_event_pointer_get_absolute_x_transformed(rawPointerEvent, (int)_displayInformation.ScreenWidthInRawPixels),
+				y: libinput_event_pointer_get_absolute_y_transformed(rawPointerEvent, (int)_displayInformation.ScreenHeightInRawPixels));
+
+			raisePointerEvent = RaisePointerMoved;
+		}
+		else if (type == LIBINPUT_EVENT_POINTER_AXIS)
+		{
+			double GetAxisValue(libinput_pointer_axis axis)
+			{
+				var source = libinput_event_pointer_get_axis_source(rawPointerEvent);
+				return source == libinput_pointer_axis_source.Wheel
+					? libinput_event_pointer_get_axis_value_discrete(rawPointerEvent, axis)
+					: libinput_event_pointer_get_axis_value(rawPointerEvent, axis);
+			}
+
+			if (libinput_event_pointer_has_axis(rawPointerEvent, libinput_pointer_axis.ScrollHorizontal) != 0)
+			{
+				properties.IsHorizontalMouseWheel = true;
+				properties.MouseWheelDelta = (int)GetAxisValue(libinput_pointer_axis.ScrollHorizontal);
+				raisePointerEvent = RaisePointerWheelChanged;
+			}
+			else if (libinput_event_pointer_has_axis(rawPointerEvent, libinput_pointer_axis.ScrollVertical) != 0)
+			{
+				properties.IsHorizontalMouseWheel = false;
+				properties.MouseWheelDelta = (int)GetAxisValue(libinput_pointer_axis.ScrollVertical);
+				raisePointerEvent = RaisePointerWheelChanged;
+			}
+		}
+		else if (type == LIBINPUT_EVENT_POINTER_BUTTON)
+		{
+			var button = libinput_event_pointer_get_button(rawPointerEvent);
+			var buttonState = libinput_event_pointer_get_button_state(rawPointerEvent);
+
+			if (buttonState == libinput_button_state.Pressed)
+			{
+				_pointerPressed.Add(button);
+
+				properties.PointerUpdateKind = button switch
+				{
+					libinput_event_code.BTN_LEFT => LeftButtonPressed,
+					libinput_event_code.BTN_MIDDLE => MiddleButtonPressed,
+					libinput_event_code.BTN_RIGHT => RightButtonPressed,
+					_ => Other
+				};
+
+				raisePointerEvent = RaisePointerPressed;
+			}
+			else
+			{
+				_pointerPressed.Remove(button);
+
+				properties.PointerUpdateKind = button switch
+				{
+					libinput_event_code.BTN_LEFT => LeftButtonReleased,
+					libinput_event_code.BTN_MIDDLE => MiddleButtonReleased,
+					libinput_event_code.BTN_RIGHT => RightButtonReleased,
+					_ => Other
+				};
+
+				raisePointerEvent = RaisePointerReleased;
+			}
+		}
+
+		properties.IsLeftButtonPressed = _pointerPressed.Contains(libinput_event_code.BTN_LEFT);
+		properties.IsMiddleButtonPressed = _pointerPressed.Contains(libinput_event_code.BTN_MIDDLE);
+		properties.IsRightButtonPressed = _pointerPressed.Contains(libinput_event_code.BTN_RIGHT);
+
+		var pointerPoint = new Windows.UI.Input.PointerPoint(
+			frameId: (uint)timestamp, // UNO TODO: How should set the frame, timestamp may overflow.
```

Comment:
Do you have an issue or task that you can link for this TODO ;) ?

Reply:
Maybe ... I just moved this code, don't know the exact status of this, and it's not part of the current PR.